src/services/hardware/hardwareObjectOptions.ts

```diff
@@ -0,0 +1,5 @@
+export interface HardwareObject {
+  name?: string;
```

Comment:
Is there any reason these fields need to be optional?

Reply:
I can't remember why they were optional. I think the only one that should be optional is the itemURL